### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by lugard1**
-  Made changes to the index.js file
-  Made changes to the style.css file
-  Made changes to the index.html file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,246 +11,142 @@
...
+      <img id="popup-img" src="./images/SnapshootPortfolio1.svg" alt="project screenshot">
+      <div id="popup-details">
+        <p id="popup-description"></p>
+        <div id="popup-skills-buttons">
+          <div id="popup-buttons">
+            <a class="enabled-btn live-link" target="_blank" rel="noopener" href="">See Live<i
+                class="fa-solid fa-arrow-up-right-from-square"></i></a>
+            <a class="enabled-btn source-link" target="_blank" rel="noopener" href="">See Source<i
+                class="fa-brands fa-github"></i></a>
+          </div>
```
Comment (Reem-lab) [inline index.html]: - [ ] Great job working on the Pop up I really appreciate your efforts but As I can see in the HTML you kept your HTML for the popup section the same but according to the requirements it is required to add dynamically like the works section so please remove all the hardcoded from the HTML and generate it using JS



image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻 

Good job so far! 💯🚀🔥🔥
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights ✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the design and functionality for dynamic js file✅
📌 Good commit messages ✅
📌 Professional README ✅




## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lugard1** [comment]
- Added popup modal dynamically

File: index.html (hunk 1 above)
Comment (lugard1) [reply index.html]: Done

**Review by mckent05 — APPROVED** [review]
Hi guys,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 

## Highlights
- Popup works well 👍 👍 
- Your code is clean 👏 
- All linter checks are passing ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**